recipe/common.php

```diff
@@ -114,13 +114,24 @@
     return locateBinaryPath('git');
 });
 
+// Composer custom version
+set('composer_version', false);
+
 set('bin/composer', function () {
     if (commandExist('composer')) {
         $composer = '{{bin/php}} ' . locateBinaryPath('composer');
     }
 
     if (empty($composer)) {
-        run("cd {{release_path}} && curl -sS https://getcomposer.org/installer | {{bin/php}}");
+        $installCommand = "cd {{release_path}} && curl -sS https://getcomposer.org/installer | {{bin/php}}";
+        
+        $composerVersion = get('composer_version', false);
```

Comment:
What if instead of specifying version we will put composer in `.dep` dir? This way user will not need this hack to fix it's version.

Reply:
.dep dir? release_path used. I only add version arg for pipeline.